webapp/src/Service/ImportExportService.php

```diff
@@ -48,22 +48,21 @@ public function getContestYamlData(Contest $contest): array
 
         $data = [
             'name' => $contest->getName(),
-            'short-name' => $contest->getShortname(),
-            'start-time' => Utils::absTime($contest->getStarttime(), true),
+            'short_name' => $contest->getShortname(),
```

Comment:
If we want to use the (new) CLICS format this should be `name` and `name` should be `formal_name`.